gradle-plugin/kubernetes/src/main/java/org/eclipse/jkube/gradle/plugin/AbstractJKubePlugin.java

```diff
@@ -60,7 +60,7 @@ private void configureTasks(Project project) {
               .map(taskByName(evaluatedProject))
               .filter(Objects::nonNull)
               // Cannot remove warning (object spread)
-              .forEach(t -> task.mustRunAfter(t));
+              .forEach(task::mustRunAfter);
```

Comment:
Not sure why this is failing in the Java8 pipeline:

```
[ERROR] /home/jenkins/agent/workspace/Build_Java8_PR-3626/gradle-plugin/kubernetes/src/main/java/org/eclipse/jkube/gradle/plugin/AbstractJKubePlugin.java:[63,24] incompatible types: invalid method reference

    mustRunAfter(java.lang.Object...) in org.gradle.api.Task is defined in an inaccessible class or interface
```

Reply:
Change undone.